packages/data-explorer/src/components/SearchComponent.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="input-group mb-3">
+    <input
+      type="text"
+      v-model="searchText"
+      class="form-control"
+      placeholder="Search"
```

Comment:
Maybe usei18n here?

Reply:
yes , but is not used elsewhere in data-explorer , suggest we skipt it for now and add it all at once. still some questions like do we add a new i18 namespace for explorer 2 or do we use the existing namespace , and do we add i18 data-explorer defaults to the war ( needs to done in molgenis repo i think)